compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/util/ProjectDirs.java

```diff
@@ -83,4 +83,50 @@ public static boolean containsSourceFiles(Path pkgPath) {
         }
         return sourceFiles.size() > 0;
     }
+
+    /**
+     * Checks if the source is from tests.
+     * @param sourcePath source path
+     * @param sourceRoot source root path
+     * @param pkg package name
+     * @return true if its a test source, else false
+     */
+    public static boolean isTestSource(Path sourcePath, Path sourceRoot, String pkg) {
```

Comment:
Update the comments to reflect each case clearly

Reply:
Added comments with 4e9c501